src/SystemPaths.cc

```diff
@@ -129,6 +158,15 @@ const std::list<std::string> &SystemPaths::PluginPaths()
     {
       this->AddPluginPaths(result);
     }
+    // TODO(CH3): Deprecated. Remove on tock.
+    else if (env(this->dataPtr->pluginPathEnvDeprecated, result))
```

Comment:
Since this is adding paths, I think we could support a mixed setup where users may have the old var as well as the new var

```suggestion
    if (env(this->dataPtr->pluginPathEnvDeprecated, result))
```

Reply:
https://github.com/gazebosim/gz-common/pull/365/commits/cb5f31e2958f927c0bcab1db07114fd24df82668